website/docs: change azure ad to entra id (#15691)
goauthentik/authentik: website/docs/users-sources/sources/social-logins/entra-id/index.mdx

---
title: Entra ID
support_level: community
---

## Preparation

The following placeholders are used in this guide:

- `authentik.company` is the FQDN of the authentik install.

## Entra ID configuration

> 1. Log in to [Entra ID](https://entra.microsoft.com) using a global administrator account.

Reviewer: ```suggestion
1. Log in to [Entra ID](https://entra.microsoft.com) using a [global administrator](https://learn.microsoft.com/en-us/entra/identity/role-based-access-control/permissions-reference#global-administrator) account.
```

2. Navigate to **Applications** > **App registrations**.
3. Click **New registration** and select the following required configurations:
    - **Name**: provide a descriptive name (e.g. `authentik`).
    - Under **Supported account types**: select the account type that applies to your use-case (e.g. `Accounts in this organizational directory only (Default Directory only - Single tenant)`).
    - Under **Redirect URI**:
        - **Platform**: `Web`
        - **URI**: `https://authentik.company/source/oauth/callback/entra-id/

4. Click **Register**.

>     - The **Overview** tab of the newly created authentik app registration opens. Take note of the `Application (client) ID`. If you selected `Accounts in this organizational directory only (Default Directory only - Single tenant)` as the **Supported account types**, also note the `Directory (tenant) ID`. These values will be required in authentik.

Reviewer: ```suggestion
    - Once the registration is complete, the **Overview** tab of the newly created authentik app opens. Take note of the `Application (client) ID`. 

If you selected `Accounts in this organizational directory only (Default Directory only - Single tenant)` as the **Supported account types**, also note the `Directory (tenant) ID`.

These values will be needed later when configuring authentik.
```

> 5. In the sidebar, navigate to **Certificates & secrets**.

Reviewer: ```suggestion
5. In the leftmost sidebar, navigate to **Certificates & secrets**.
```

Probably just to clarify

also i haven't  been in the azure portal in a while, is the second element after & lowercase or uppercase, just to be sure it matches

> 6. Select the **Client secrets** tab and click **New Secret**. Provide the following required configurations:

Reviewer: also the right term wouldn't really be "provide" i feel. mabye going for "Configure the following required settings:" or anything which doesn't have Provide in that tone. As you wish

    - **Description**: provide a description for the secret (e.g. `authentik secret`.
    - **Expires**: select an expiry duration. Currently the secret in authentik has to be rotated manually or via API, so it is recommended to choose at least 12 months.
7. Copy the secret's value from the **Value** column.

:::note
The secret value must be copied immediately after its creation. It is not possible to view the secret value later.
:::

> 9. In the sidebar, navigate to **API Permissions**.

Reviewer: could mabye be merged w step 11

> 9. In the sidebar, navigate to **API Permissions**.

Reviewer: 10*
